Formula/strace.rb

```diff
@@ -1,8 +1,8 @@
 class Strace < Formula
   desc "Diagnostic, instructional, and debugging tool for the Linux kernel"
   homepage "https://strace.io/"
-  url "https://github.com/strace/strace/archive/v4.16.tar.gz"
-  sha256 "4a6dcf63f2b3e49b840380396d2a8a5a4fb30ca12f953d82c16236bfe620e45a"
+  url "https://downloads.sourceforge.net/project/strace/strace/4.17/strace-4.17.tar.xz"
```

Comment:
The GitHub releases page warns that the GitHub source code links are "useless". Not sure why the author thinks that, but I won't argue.